java/code/src/com/redhat/rhn/frontend/xmlrpc/activationkey/ActivationKeyHandler.java

```diff
@@ -810,10 +798,10 @@ public int addPackages(User loggedInUser, String key,
      * @xmlrpc.param #param("string", "sessionKey")
      * @xmlrpc.param #param("string", "key")
      * @xmlrpc.param
-     *   #array_begin()
-     *      #struct_begin("packages")
-     *          #prop_desc("string", "name", "Package name")
-     *          #prop_desc("string", "arch", "Arch label - Optional")
+     *   #array_begin("packages")
+     *     #struct_begin("pacakge")
```

Comment:
typo

Reply:
Not really. The important one is "packages", which is the parameter name. Nested struct name "package" is just for cosmetics. It's singular because it describes a single item in the array.

Nested serializers add similar descriptions automatically.

Reply:
What I mean is `pacakge` -> `package`